www/pages/blog/[slug].tsx

```diff
@@ -0,0 +1,292 @@
+// file: pages/blog/[slug].js
```

Comment:
@MildTomato @inian some thoughts:

Since we are unsure of the monorepo setup, perhaps for now it might be better to keep the exact same URL structure? 

If you agree, the possible changes:

- rename this file to be `/pages/blog/[year]/[month]/[date]/[slug].tsx`
- Remove all the redirects in `vercel.json`

We can revisit the URLs in the future when the team has more time

Reply:
yea sure.

might also need to update lib/posts as well so it gets the right post.

Reply:
Yup let's do that Copple.

Reply:
I'll sort this out

Reply:
updated post urls.

have to pop out for an hour, will update links to blog posts then can merge

Reply:
done